Makefile

```diff
@@ -133,15 +137,13 @@ lint:
 	# GO111MODULE=on revive -config revive.toml -exclude storage/ledger/trie ./...
 	golangci-lint run -v --build-tags relic ./...
 
-# Runs linter, unit tests, SKIP FOR NOW coverage
+# Runs unit tests, SKIP FOR NOW linter, coverage
 .PHONY: ci
-ci: install-tools tidy lint test # coverage
+ci: clean-relic install-tools tidy test # lint coverage
 
 # Runs integration tests
-# NOTE: we do not need `docker-build-flow` as this is run as a separate step
-# on Teamcity
 .PHONY: ci-integration
-ci-integration: install-tools
+ci-integration: crypto/relic/build
```

Comment:
Was the dependency on `install-tools` removed because linting is now a separate job?

Reply:
yup, one of the key reasons